### DISCUSSION TITLE
Weird color in Rio

### DISCUSSION THREAD

**Discussion by linsui** [sxyazi/yazi#771] [1. Q&A]
![图片](https://github.com/sxyazi/yazi/assets/36977733/17057cf5-016c-42d8-a368-d5db59bb95e2)
This is my theme in Rio. It's the same as in Blackbox. However the color yazi used in Rio is different.

In Blackbox:

![图片](https://github.com/sxyazi/yazi/assets/36977733/c19302a7-7d84-408f-a9f0-0ae265ca72a2)

In Rio:

![图片](https://github.com/sxyazi/yazi/assets/36977733/878b82fb-c3ed-4f9a-8eb5-316e182aa32e)

You can see that the color yazi uses doesn't exist in my theme. I don't know where it comes from.

**Comment by sxyazi** [comment]
If you're using the default theme of Yazi, which uses standard ANSI 16 colors, it's possible that Rio is incorrectly parsing them.

This might be related to this issue: https://github.com/raphamorim/rio/issues/464

**Reply by linsui** [reply]
Wow, I thought your are right, thanks!
